.docker/nginx_default.conf

```diff
@@ -0,0 +1,34 @@
+server {
```

Comment:
With the removal of the nginx fpm container is this configuration file still needed?